### PR TITLE
fixed how to get packages in develop

### PR DESCRIPTION
**Pull request by eikichi18**
it only worked with a package under development

### CODE REVIEW

**Review by Rudd-O — APPROVED** [review]

**Comment by jensens** [comment]
This is probably not the end of the development here and with upcoming developments planned might be obsolete at some point in the future, but for now its good to have.

**Review by jensens — APPROVED** [review]

**Comment by stevepiercy** [comment]
Does this change anything in the Plone 6 docs, specifically https://6.docs.plone.org/install/manage-add-ons-packages.html#common-management-tasks ?

**Comment by jensens** [comment]
Another place, but, indeed, this change is not covered by this chapter here https://6.docs.plone.org/install/containers/images/backend.html#add-on-variables and the part here https://6.docs.plone.org/install/containers/images/backend.html#containers-images-backend-developing-packages-label -  there is an update needed.

File: skeleton/docker-entrypoint.sh [hunk 1]
```diff
@@ -113,7 +113,13 @@ if [[ -v DEVELOP ]]; then
...
   echo "Read about it: https://6.docs.plone.org/install/containers/images/backend.html"
   echo "======================================================================================="
-  $VENVBIN/pip install --editable ${DEVELOP} ${PIP_PARAMS}
+  PACKAGES=""
+  while read -ra DEVADDONS; do
+    for a in "${DEVADDONS[@]}"; do
+      PACKAGES+=" --editable ${a}"
+    done
+  done <<< "$DEVELOP"
+  $VENVBIN/pip install ${PACKAGES} ${PIP_PARAMS}
```
Comment (Rudd-O) [inline skeleton/docker-entrypoint.sh:122]: Nice.

**Comment by stevepiercy** [comment]
Issue create in `plone/documentation`. https://github.com/plone/documentation/issues/1434

I would prefer that pull requests that require changes to documentation be held up until there is at least a pull request created in `plone/documentation`.

**Pull request merged**